.stylelintrc.json

```diff
@@ -4,6 +4,7 @@
 	"root": true,
   "rules": {
     "prettier/prettier": true,
-    "no-eol-whitespace": true
+    "no-eol-whitespace": true,
+    "selector-class-pattern": null
```

Comment:
I see the disadvantage that custom styles might not work anymore, but nonetheless I would like to transform the names to kebab-case. Then we would not need to disable this rule. What do you think?

Reply:
Maybe doing the transformation should be done in another PR after this is merged (and when @deg0nz weighted in on this)?